sdk/core/azure-core-tracing-opentelemetry/src/main/java/com/azure/core/tracing/opentelemetry/OpenTelemetryHttpPolicy.java

```diff
@@ -95,21 +100,21 @@ public HttpPipelinePolicy create() {
 
     private static void addSpanRequestAttributes(Span span, HttpRequest request,
         HttpPipelineCallContext context) {
-        putAttributeIfNotEmptyOrNull(span, SemanticAttributes.HTTP_USER_AGENT,
+        putAttributeIfNotEmptyOrNull(span, HTTP_USER_AGENT,
             request.getHeaders().getValue("User-Agent"));
-        putAttributeIfNotEmptyOrNull(span, SemanticAttributes.HTTP_METHOD, request.getHttpMethod().toString());
-        putAttributeIfNotEmptyOrNull(span, SemanticAttributes.HTTP_URL, request.getUrl().toString());
+        putAttributeIfNotEmptyOrNull(span, HTTP_METHOD, request.getHttpMethod().toString());
+        putAttributeIfNotEmptyOrNull(span, HTTP_URL, request.getUrl().toString());
         Optional<Object> tracingNamespace = context.getData(AZ_TRACING_NAMESPACE_KEY);
         if (tracingNamespace.isPresent()) {
-            putAttributeIfNotEmptyOrNull(span, AttributeKey.stringKey(OpenTelemetryTracer.AZ_NAMESPACE_KEY),
+            putAttributeIfNotEmptyOrNull(span, OpenTelemetryTracer.AZ_NAMESPACE_KEY,
                 tracingNamespace.get().toString());
         }
     }
 
-    private static void putAttributeIfNotEmptyOrNull(Span span, AttributeKey<String> key, String value) {
+    private static void putAttributeIfNotEmptyOrNull(Span span, String key, String value) {
         // AttributeValue will throw an error if the value is null.
```

Comment:
Will this also throw on an empty string value too? I wonder if the conditional below should just check `!= null`

Reply:
>Will this also throw on an empty string value too?
It won't throw but just silently ignore that attr value.

I don't see much value in setting empty attribute values.